bin/compile

```diff
@@ -1,15 +1,15 @@
 #!/usr/bin/env bash
 # bin/compile <build-dir> <cache-dir>
 
-echo "-----> Installing Ghostscript 9.53.3"
+echo "-----> Installing Ghostscript 9.54.0"
 
 BUILD_DIR=$1
-PACKAGE="https://s3.amazonaws.com/tgl.public/buildpacks/ghostscript/ghostscript-9.53.3-linux-x86_64.tgz"
```

Comment:
@bvirlet do you remember why we kept the GS binary on S3? Just in case Artifex server is down at some point?

Reply:
Artifex is the maintainer of Ghostscript. It is unlikely to be down. However, I can move the release to our S3 if you grant me write access. :-)

Reply:
Let's leave it like this for now.